test/bdd/fixtures/agent-wasm/docker-compose.yml

```diff
@@ -125,18 +146,29 @@ services:
       - hydra
       - mock-bddtest-login-consent.example.com
 
+  bddtest.ui.user.agent.example.com:
+    container_name: bddtest.ui.user.agent.example.com
+    image: ${AGENT_UI_IMAGE}:latest
+    volumes:
+      - ../keys/tls:/etc/nginx/certs
+    ports:
+      - 8078:443
+    environment:
+      - ARIESD_HTTP_RESOLVER=${HTTP_RESOLVER}
+      - AGENT_DEFAULT_LABEL=user-agent
+      - BLOC_DOMAIN=testnet.trustbloc.local
+      - EDGE_AGENT_SERVER=https://localhost:8077
+      - WALLET_MEDIATOR_URL=${WALLET_ROUTER_URL}
+
   bddtest.user.agent.example.com:
     container_name: bddtest.user.agent.example.com
     image: ${USER_WASM_IMAGE}:latest
     environment:
+      - AGENT_UI_URL=https://localhost:8078
```

Comment:
@soluchok Looks like we have cyclic dependency - agent server calling web-wallet-ui and [vice versa](https://github.com/trustbloc/edge-agent/pull/492/files#diff-0325567f17eab8064d99ba9a4ac40c08a40d423010c14d597c0ab8eae6e62ce1R160).